### PR TITLE
indexer version

### PR DESCRIPTION
**Pull request by yingjieMiao**
@ymatsuda @leogrim

### CODE REVIEW

File: kifi-backend/modules/search/app/com/keepit/search/index/IndexerVersion.scala [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+object IndexerVersion {
+  implicit def toInt(version: IndexerVersion): Int = version.value
+  implicit def fromInt(value: Int) = IndexerVersion(value)
+}
+
+abstract class IndexerVersionProvider(activeVersion: IndexerVersion, backupVersion: IndexerVersion) {
+  require(backupVersion >= activeVersion)
+  def getVerionByStatus(service: ServiceDiscovery): IndexerVersion = if (service.isBackup) backupVersion else activeVersion
```
Comment (leogrim) [inline kifi-backend/modules/search/app/com/keepit/search/index/IndexerVersion.scala]: small typo

**Comment by leogrim** [comment]
Awesome!

File: kifi-backend/modules/search/app/com/keepit/search/index/IndexerVersion.scala (hunk 1 above)
Comment (yingjieMiao) [inline kifi-backend/modules/search/app/com/keepit/search/index/IndexerVersion.scala]: nice catch!

**Comment by ymatsuda** [comment]
Nice!

**Pull request merged**